x-pack/elastic-agent/pkg/agent/application/gateway/fleet/fleet_gateway.go

```diff
@@ -127,7 +127,7 @@ func newFleetGatewayWithScheduler(
 	stateStore stateStore,
 ) (gateway.FleetGateway, error) {
 
-	// Backoff implementation doesn't support the using context as the shutdown mechanism.
+	// Backoff implementation doesn't support the using of a context as the shutdown mechanism.
```

Comment:
```suggestion
	// Backoff implementation doesn't support the using cancellation of a context as the shutdown mechanism.
```
Is this what is meant?

Reply:
Suggest either "Backoff implementation doesn't support using a context [cancellation] as the shutdown mechanism." or "Backoff implementation doesn't support the use of a context [cancellation] as the shutdown mechanism." (insert "cancellation" according to taste).